nl.emphisia.icon.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "nl.emphisia.icon",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```

Comment:
there is no such runtime version.

Reply:
Oh, weird. I will take a long look at what I am doing wrong very soon. thank you very much!